network/network_windows.go

```diff
@@ -302,6 +302,11 @@ func (nm *networkManager) configureHcnNetwork(nwInfo *EndpointInfo, extIf *exter
 		hcnNetwork.Flags = hcn.DisableHostPort
 	}
 
+	if nwInfo.NICType == cns.NodeNetworkInterfaceAccelnetFrontendNIC {
+		hcnNetwork.Type = hcn.Transparent
+		hcnNetwork.Flags = 9216 // temp hardcode until hcsshim supports the flag
```

Comment:
isn't `hcn.DisableHostPort` not required for this?

Reply:
I will use another PR for accelnet NIC	
	if nwInfo.NICType == cns.NodeNetworkInterfaceAccelnetFrontendNIC {
		hcnNetwork.Flags = hcn.DisableHostPort | 8192 // temp hardcode until hcsshim supports the flag